components/scream/src/physics/zm/scream_zm_interface.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef SCREAM_ZM_INTERFACE_HPP
+#define SCREAM_ZM_INTERFACE_HPP
+
+#include "ekat/scream_assert.hpp"
```

Comment:
These files changed name in a recent PR merge where we updated the version of the ekat submodule. You will have to merge master in your branch...and cross your fingers.

Jokes aside, if you get too many conflicts and need help, or are note sure on how to resolve some of them, let me know.